tests/integration/suite/daprd/workflow/purge/retention/config/match/terminated.go

```diff
@@ -0,0 +1,126 @@
+/* Copyright 2025 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://wwb.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"context"
+	"errors"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/dapr/dapr/tests/integration/framework"
+	"github.com/dapr/dapr/tests/integration/framework/process/daprd"
+	"github.com/dapr/dapr/tests/integration/framework/process/workflow"
+	"github.com/dapr/dapr/tests/integration/suite"
+	dworkflow "github.com/dapr/durabletask-go/workflow"
+)
+
+func init() {
+	suite.Register(new(terminated))
+}
+
+type terminated struct {
+	workflow *workflow.Workflow
+}
+
+func (e *terminated) Setup(t *testing.T) []framework.Option {
+	e.workflow = workflow.New(t,
+		workflow.WithDaprdOptions(0, daprd.WithConfigManifests(t, `apiVersion: dapr.io/v1alpha1
+kind: Configuration
+metadata:
+  name: wfpolicy
+spec:
+  workflow:
+    stateRetentionPolicy:
+      anyTerminal: "5s"
+      terminated: "0s"
+`)),
+	)
+
+	return []framework.Option{
+		framework.WithProcesses(e.workflow),
+	}
+}
+
+func (e *terminated) Run(t *testing.T, ctx context.Context) {
```

Comment:
I think it's worth adding a note of what this test is about, it took me some time to figure out it's about the configuration having both `anyTerminal` and `terminated`.